### ISSUE TITLE
url publique

### ISSUE THREAD

**Issue by Tstassin** [Samuel85sam/Template-mantine#1]
Hello Sam, ton CV est-il publié quelque part ?

**Comment by Samuel85sam** [comment]
non, pas pour le moment. Je te l'ai renvoyé par mail aussi au cas où... à bientôt
[C.V. Samuel De Mees.pdf](https://github.com/Samuel85sam/Template-mantine/files/15106914/C.V.Samuel.De.Mees.pdf)
